tests/05_read_lab_time_test.py

```diff
@@ -0,0 +1,34 @@
+"""Test read_lab_time.
+
+This module tests the correctness and exceptions of ReadCourseData/read_lab_time()
+"""
+
+import sys
+sys.path.append('./src')
+from InsertData import check_file_open
+from ReadCourseData import read_lab_time
+import course_pb2 as course
+
+def test_read_lab_time_correctness_lab():
+    """Test if read_lab_time() returns the right content if the course has a lab."""
+    file_obj = check_file_open('tests/test_lab.json')
+    each_course = file_obj['Test Data']['CourseData']['ACCT'][0]
+    temp_course = course.Course()
+    temp_course = read_lab_time(each_course, temp_course)
+    assert len(temp_course.lab) == 1
+    assert temp_course.lab[0].UID == 'L'
+    assert temp_course.lab[0].days == ''
+    assert temp_course.lab[0].time == 'TBA'
+    assert temp_course.lab[0].startDate == '06/28'
+    assert temp_course.lab[0].endDate == '08/07'
+    assert temp_course.lab[0].instructor == 'Christopher,C,Kwak'
+    assert temp_course.lab[0].location == 'DA LC107'
```

Comment:
Should we use a list of target results and loop through it?

Reply:
Yes